### ISSUE TITLE
Remove Backspace functionality

### ISSUE THREAD

**Issue by AndreyNazarchuk** [phpmyadmin/phpmyadmin#14159]
Make backspace not move to the previous page. It's annoying and no one actually uses it intentionally. The backspace to go back has already been disabled in modern browsers. Why did you guys add it back.

**Comment by ibennetch** [comment]
Are you sure this is something phpMyAdmin is adding? I've tested in several browsers, including Chrome and Firefox. In Chrome and Safari, backspace doesn't do anything even in phpMyAdmin; with Firefox backspace does go back a page, but it's on any page and doesn't seem to be related to anything phpMyAdmin is doing.

**Comment by OlafvdSpek** [comment]
> and no one actually uses it intentionally. 

Excuse me?

**Comment by AndreyNazarchuk** [comment]
@OlafvdSpek I have never in my life used it or wanted to use it to go back a page. It often deletes a long query I was still using. 

@ibennetch it only happens on the query page if you're not actively in a text box.

**Comment by OlafvdSpek** [comment]
@AndreyNazarchuk What browser?
I use Backspace all the time, especially when my mouse doesn't have a back button.

**Comment by AndreyNazarchuk** [comment]
@OlafvdSpek 
windows 10, Google Chrome

I tried looking through the settings and installing backspace as back disablers, but nothing seems to be working.

Please add an option in the settings to disable it.

**Comment by AndreyNazarchuk** [comment]
@ibennetch 
To reproduce go to a table and type a query and hit go. Then hit backspace.

**Comment by OlafvdSpek** [comment]
Backspace as back is disabled by default in Chrome.

**Comment by williamdes** [comment]
> To reproduce go to a table and type a query and hit go. Then hit backspace.

Confirmed on chrome.
https://demo.phpmyadmin.net/QA_4_8/tbl_sql.php?db=menagerie&table=event
https://demo.phpmyadmin.net/master-config/tbl_sql.php?db=menagerie&table=event

**Comment by zzc1231** [comment]
use my Iphone I can`t delete wrong Input，it will go back .

**Comment by stuartcusackie** [comment]
Yep. I'm using phpmyadmin on a local virtual server and the browser goes back when I try to delete text in a table row.  This doesn't happen on any other website. Weird.

**Comment by Toopala** [comment]
At least add setting to disable the functionality. This is by far the most infuriating thing about PHPmyadmin. I have big database and I navigate it by the tree view and filter the tables via the textbox. When you hit enter, the focus on the textbox is lost. If I made a typo or mistakenly wrote the wrong table I instinctively press backspace to correct myself and see myself back on the login screen.

**Comment by patchlog** [comment]
I'm using the latest version of firefox and this doesn't happen with any other site but in phpmyadmin if I'm not fucused in a text field or textarea and I hit backspace ( delete on a mac ) it takes me back to the previous page. This is so annoying.

**Comment by OlafvdSpek** [comment]
It happens on Chrome / W10 too..

> if I'm not fucused in a text field or textarea and I hit backspace ( delete on a mac ) it takes me back to the previous page. This is so annoying.

Why are you hitting Delete? ;)

**Comment by patchlog** [comment]
> It happens on Chrome / W10 too..
> 
> > if I'm not fucused in a text field or textarea and I hit backspace ( delete on a mac ) it takes me back to the previous page. This is so annoying.
> 
> Why are you hitting Delete? ;)
I try to delete something in a query but maybe tapped on the touchpad by mistake and the text field was defocused.
